src/webui/webapplication.cpp

```diff
@@ -224,12 +232,15 @@ void WebApplication::action_public_images()
 void WebApplication::action_query_torrents()
 {
     CHECK_URI(0);
-    const QStringMap& gets = request().gets;
+    const RawStringMap& gets = request().gets;
 
-    print(btjson::getTorrents(
-        gets["filter"], gets["category"], gets["sort"], gets["reverse"] == "true",
-        gets["limit"].toInt(), gets["offset"].toInt()
-        ), Http::CONTENT_TYPE_JSON);
+    print(btjson::getTorrents( toUtf8(gets["filter"]),
```

Comment:
`fromUtf8()` only makes sense for the category. All other parameters have been designed to contain either numbers or latin-1 string. But you may leave it.

Also fix coding style in this line.